syz-cluster/pkg/blob/storage.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2024 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package blob
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+// Storage is not assumed to be used for partciularly large objects (e.g. GB of size),
+// but rather for blobs that risk overwhelming Spanner column size limits.
+type Storage interface {
```

Comment:
What do you plan to store here?
I have a feeling we can reuse existing components/systems. Gcs? Redis? Spanner?
What do you mean by "overwhelming Spanner column size limits"?